packages/-ember-data/yuidoc.json

```diff
@@ -13,7 +13,9 @@
       "../serializer/addon",
       "../store/addon",
       "../record-data/addon",
-      "../debug/addon"
+      "../debug/addon",
+      "../private-build-infra/addon",
```

Comment:
Why are we adding the private-build infra to yuidocs?

Reply:
we need to parse to get the `@ember-data/deprecations` guide